code/__HELPERS/_macros.dm

```diff
@@ -5,8 +5,30 @@
 // fun if you want to typecast humans/monkeys/etc without writing long path-filled lines.
 #define ishuman(A) istype(A, /mob/living/carbon/human)
 
+#define isjusthuman(A) A.type == /mob/living/carbon/human
+
 #define ismonkey(A) istype(A, /mob/living/carbon/monkey)
 
+#define isvox(A) istype(A, /mob/living/carbon/human/vox)
```

Comment:
Are macros limited to whatever you can put into an if statement?
I could probably make a proc that would do the casting and comparing and then just have the macro access it.